tools/nni_cmd/tests/mock/nnictl_metadata/.experiment

```diff
@@ -0,0 +1 @@
+{"xOpEwA5w": {"port": 8080, "startTime": "1970/01/1 01:01:01", "endTime": "1970-01-2 01:01:01", "status": "RUNNING", "fileName": "aGew0x", "platform": "local", "experimentName": "example_sklearn-classification"}}
```

Comment:
do we need to checkin this file?

Reply:
yes, this is a mock data to test nnictl read config functions.